diff --git a/Cargo.toml b/Cargo.toml
index b787a012f..7df4ec59c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4967,18 +4967,6 @@ rustdoc-args = [
 ]
 all-features = true
 
-[[example]]
-name = "monitor_info"
-path = "examples/window/monitor_info.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.monitor_info]
-name = "Monitor info"
-description = "Displays information about available monitors (displays)."
-category = "Window"
-wasm = false
-
 # Testbed
 [[example]]
 name = "testbed_2d"
