diff --git a/Cargo.toml b/Cargo.toml
index dc0cf74f0..a4e034119 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1932,17 +1932,6 @@ description = "Creates and customizes the internal thread pool"
 category = "Application"
 wasm = false
 
-[[example]]
-name = "no_renderer"
-path = "examples/app/no_renderer.rs"
-doc-scrape-examples = true
-
-[package.metadata.example.no_renderer]
-name = "No Renderer"
-description = "An application that runs with default plugins and displays an empty window, but without an actual renderer"
-category = "Application"
-wasm = false
-
 [[example]]
 name = "externally_driven_headless_renderer"
 path = "examples/app/externally_driven_headless_renderer.rs"
