diff --git a/Cargo.toml b/Cargo.toml
index e19c42cdb..f7111c544 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4670,17 +4670,6 @@ rustdoc-args = [
 ]
 all-features = true
 
-[[example]]
-name = "monitor_info"
-path = "examples/window/monitor_info.rs"
-doc-scrape-examples = true
-
-[package.metadata.example.monitor_info]
-name = "Monitor info"
-description = "Displays information about available monitors (displays)."
-category = "Window"
-wasm = false
-
 # Testbed
 [[example]]
 name = "testbed_2d"
