diff --git a/Cargo.toml b/Cargo.toml
index 9592d2d1e..19799ba48 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4800,54 +4800,6 @@ description = "Demonstrates volumetric fog and lighting"
 category = "3D Rendering"
 wasm = true
 
-[[example]]
-name = "client"
-path = "examples/remote/client.rs"
-doc-scrape-examples = true
-required-features = ["bevy_remote"]
-
-[package.metadata.example.client]
-name = "client"
-description = "A simple command line client that can control Bevy apps via the BRP"
-category = "Remote Protocol"
-wasm = false
-
-[[example]]
-name = "server"
-path = "examples/remote/server.rs"
-doc-scrape-examples = true
-required-features = ["bevy_remote"]
-
-[package.metadata.example.server]
-name = "server"
-description = "A Bevy app that you can connect to with the BRP and edit"
-category = "Remote Protocol"
-wasm = false
-
-[[example]]
-name = "app_under_test"
-path = "examples/remote/app_under_test.rs"
-doc-scrape-examples = true
-required-features = ["bevy_remote"]
-
-[package.metadata.example.app_under_test]
-name = "App Under Test"
-description = "A Bevy app that you can connect to with the BRP and control"
-category = "Remote Protocol"
-wasm = false
-
-[[example]]
-name = "integration_test"
-path = "examples/remote/integration_test.rs"
-doc-scrape-examples = true
-required-features = ["bevy_remote"]
-
-[package.metadata.example.integration_test]
-name = "Integration Test"
-description = "Connects to a running Bevy app via BRP, finds a button, and clicks it"
-category = "Remote Protocol"
-wasm = false
-
 [[example]]
 name = "anisotropy"
 path = "examples/3d/anisotropy.rs"
