diff --git a/Cargo.toml b/Cargo.toml
index b787a012f..1b3680319 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4741,30 +4741,6 @@ description = "Demonstrates how to enqueue custom draw commands in a render phas
 category = "Shaders"
 wasm = true
 
-[[example]]
-name = "fog_volumes"
-path = "examples/3d/fog_volumes.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.fog_volumes]
-name = "Fog volumes"
-description = "Demonstrates fog volumes"
-category = "3D Rendering"
-wasm = false
-
-[[example]]
-name = "scrolling_fog"
-path = "examples/3d/scrolling_fog.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.scrolling_fog]
-name = "Scrolling fog"
-description = "Demonstrates how to create the effect of fog moving in the wind"
-category = "3D Rendering"
-wasm = false
-
 [[example]]
 name = "physics_in_fixed_timestep"
 path = "examples/movement/physics_in_fixed_timestep.rs"
