diff --git a/Cargo.toml b/Cargo.toml
index b787a012f..788348d11 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1308,18 +1308,6 @@ description = "Decal rendering"
 category = "3D Rendering"
 wasm = true
 
-[[example]]
-name = "deferred_rendering"
-path = "examples/3d/deferred_rendering.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.deferred_rendering]
-name = "Deferred Rendering"
-description = "Renders meshes with both forward and deferred pipelines"
-category = "3D Rendering"
-wasm = true
-
 [[example]]
 name = "motion_blur"
 path = "examples/3d/motion_blur.rs"
