diff --git a/Cargo.toml b/Cargo.toml
index b787a012f..073183ab3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4679,18 +4679,6 @@ description = "Demonstrates depth of field"
 category = "3D Rendering"
 wasm = false
 
-[[example]]
-name = "volumetric_fog"
-path = "examples/3d/volumetric_fog.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.volumetric_fog]
-name = "Volumetric fog"
-description = "Demonstrates volumetric fog and lighting"
-category = "3D Rendering"
-wasm = true
-
 [[example]]
 name = "client"
 path = "examples/remote/client.rs"
