diff --git a/Cargo.toml b/Cargo.toml
index b787a012f..4ba12d635 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4618,18 +4618,6 @@ description = "Demonstrates visibility ranges"
 category = "3D Rendering"
 wasm = true
 
-[[example]]
-name = "ssr"
-path = "examples/3d/ssr.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.ssr]
-name = "Screen Space Reflections"
-description = "Demonstrates screen space reflections with water ripples"
-category = "3D Rendering"
-wasm = false
-
 [[example]]
 name = "camera_sub_view"
 path = "examples/3d/camera_sub_view.rs"
