diff --git a/Cargo.toml b/Cargo.toml
index 690aad88b..afd7ddfc4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4895,18 +4895,6 @@ description = "Demonstrates depth of field"
 category = "3D Rendering"
 wasm = false
 
-[[example]]
-name = "volumetric_fog"
-path = "examples/3d/volumetric_fog.rs"
-# Causes an ICE on docs.rs
-doc-scrape-examples = false
-
-[package.metadata.example.volumetric_fog]
-name = "Volumetric fog"
-description = "Demonstrates volumetric fog and lighting"
-category = "3D Rendering"
-wasm = true
-
 [[example]]
 name = "anisotropy"
 path = "examples/3d/anisotropy.rs"
