diff --git a/Cargo.toml b/Cargo.toml
index 27470ce17..08df2d124 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1307,17 +1307,6 @@ description = "Decal rendering"
 category = "3D Rendering"
 wasm = true
 
-[[example]]
-name = "deferred_rendering"
-path = "examples/3d/deferred_rendering.rs"
-doc-scrape-examples = true
-
-[package.metadata.example.deferred_rendering]
-name = "Deferred Rendering"
-description = "Renders meshes with both forward and deferred pipelines"
-category = "3D Rendering"
-wasm = true
-
 [[example]]
 name = "motion_blur"
 path = "examples/3d/motion_blur.rs"
